app/tests/test_core/test_grpc_connection.py

```diff
@@ -15,12 +15,16 @@
 from google.protobuf.json_format import MessageToDict, MessageToJson
 import os
 
-GRPC_HOST=os.getenv('GRPC_HOST','localhost')
-GRPC_PORT=os.getenv('GRPC_PORT',50051)
+GRPC_HOST = os.getenv("GRPC_HOST", "localhost")
+GRPC_PORT = os.getenv("GRPC_PORT", 50051)
```

Comment:
Does this needs to be consistent with  https://github.com/Ensembl/ensembl-web-metadata-api/blob/coding_stats/app/api/resources/metadata.py#L26 ?